packages/lesswrong/components/ultraFeed/UltraFeedCommentItem.tsx

```diff
@@ -211,13 +258,34 @@ export const UltraFeedCommentItem = ({
   parentAuthorName,
   onReplyIconClick,
   isHighlightAnimating = false,
+  replyConfig,
+  hasFork,
+  currentBranch,
+  onBranchToggle,
+  cannotReplyReason: customCannotReplyReason,
 }: UltraFeedCommentItemProps) => {
   const classes = useStyles(styles);
   const { observe, unobserve, trackExpansion, hasBeenLongViewed, subscribeToLongView, unsubscribeFromLongView } = useUltraFeedObserver();
   const elementRef = useRef<HTMLDivElement | null>(null);
   const { openDialog } = useDialog();
   const overflowNav = useOverflowNav(elementRef);
-  const { displayStatus } = metaInfo;
+  const currentUser = useCurrentUser();
+  
+  const cannotReplyReason = customCannotReplyReason ?? (userOwns(currentUser, comment) ? "You cannot reply to your own comment from within the feed" : null);
+
+  // Provide default metaInfo if not provided
+  const safeMetaInfo = metaInfo || {
```

Comment:
nit: prefer `??`.  (also, is `metaInfo` ever going to be null/undefined?  it doesn't seem like it's typed that way in the component props...)

Reply:
(also, if it sometimes undefined and you want to assign a default in those cases, I'd prefer to extract it to a constant outside the component and assign it as a default in the prop destructuring)